.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: c++
+sudo: false
+
+addons:
+  apt:
+    packages:
+      - build-essential
+      - git
+      - cmake
+      - libboost-all-dev
+      - libxml-xpath-perl
+      - omniorb
+      - omniidl
+      - omniorb-idl
+      - omniorb-nameserver
+      - libomniorb4-dev
+
+matrix:
+    include:
+        - os: linux
+          dist: trusty
+        - os: osx
+          osx_image: xcode7.3
+        - os: osx
+          osx_image: xcode8.2
+
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew reinstall boost cmake omniorb gnu-getopt ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew link --force gnu-getopt ; fi
+
+script:
+  - ./install.sh --verbose --prefix $HOME/install --enable-tests --enable-corba --enable-mqueue
```

Comment:
We don't actually run the tests, so why build it? Or should we add another line that runs the tests after installation? Or would you expect that `--enable-tests` not only builds but also runs the tests?